### DISCUSSION TITLE
The data types were mixed for odd schemas

### DISCUSSION THREAD

**Discussion by fernandodelago** [dbeaver/dbeaver#19510] [DBeaver Development]
Hi there.

I noticed when I was working on a project with SQL Server, that the Dbeaver is have been shown all the data types in all sections "Datatypes" in all schemas. I got this when creating 2 datatypes with the same name in different schemas.
After finishing the creation, I checked in the datatype section, and to my surprise, I found the 2 datatypes together (the original datatype in that schema and the other that was owned from another schema). Take a look at picture dbeaver01.jpg.
The curious is when you open the datatype, the schema is not added in the DDL. Take a look at picture dbeaver02.jpg.
And, when I checked the other schema the same datatype was shown in that too. Take a look at picture dbeaver03.jpg.
In my mind, each schema should have shown only your datatypes, right?
![dbeaver01](https://user-images.githubusercontent.com/39306731/228172807-7b1d6f97-71b4-4e3f-935f-ff54270d5ed3.jpg)
![dbeaver02](https://user-images.githubusercontent.com/39306731/228172845-f57832e3-06eb-4d80-b076-91d1405d1449.jpg)
![dbeaver03](https://user-images.githubusercontent.com/39306731/228172869-85efb5d2-fb67-4ca7-93a0-9dea625fb821.jpg)

**Comment by LonwoLonwo** [comment]
Yeah, incorrect JOIN. Thanks for the bug report @fernandodelago
